Index: src/test/java/pages/FeaturesPage.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/pages/FeaturesPage.java	(date 1531805522484)
+++ src/test/java/pages/FeaturesPage.java	(date 1531805522484)
@@ -0,0 +1,50 @@
+package pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+public class FeaturesPage extends  Utils.Base
+{
+    // default feature Tab
+    @FindBy(how = How.XPATH, using = "//*[@id=\"wizard_menu\"]/a[1]/span")
+    public WebElement appFeaturesTab;
+    // style Tab
+    @FindBy(how = How.XPATH, using = "//*[@id=\"wizard_menu\"]/a[2]/span")
+    public WebElement styleAndNavigationTab;
+    // pluse icon on the top right corner
+    @FindBy(how = How.XPATH, using = "//*[@id=\"my_pages\"]/div/a[1]")
+    public WebElement plusIcon;
+    // left arrow
+    @FindBy(how = How.XPATH, using = "//*[@id=\"my_pages\"]/div/a[2]")
+    public WebElement navigateLeft;
+    // right arrow
+    @FindBy(how = How.XPATH, using = "//*[@id=\"my_pages\"]/div/a[3]")
+    public WebElement navigateRight;
+
+    @FindBy(how = How.XPATH, using = "//*[@id=\"pages_list_container\"]/ul/li[1]/a/span[2]")
+    public WebElement feature_1;
+
+    @FindBy(how = How.XPATH, using = "//*[@id=\"pages_list_container\"]/ul/li[2]/a/span[2]")
+    public WebElement feature_2;
+
+    @FindBy(how = How.XPATH, using = "//*[@id=\"pages_list_container\"]/ul/li[3]/a/span[2]")
+    public WebElement feature_3;
+
+    @FindBy(how = How.XPATH, using = "//*[@id=\"pages_list_container\"]/ul/li[4]/a/span[2]")
+    public WebElement feture_4;
+
+    @FindBy(how = How.XPATH, using = "//*[@id=\"pages_list_container\"]/ul/li[5]/a")
+    public WebElement feature_5;
+    //add feature button
+    @FindBy(how = How.XPATH, using = "//*[@id=\"editor\"]/div[6]/div/a")
+    public WebElement addFeatureButton;
+
+  // need to continue the page elemnts identification inside the features!
+
+
+
+
+
+
+}
