Index: src/test/java/Tests/testingElements.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/Tests/testingElements.java	(date 1534880857045)
+++ src/test/java/Tests/testingElements.java	(date 1534880857045)
@@ -0,0 +1,51 @@
+package Tests;
+
+import Utils.Base;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+public class testingElements extends Base
+{
+    @BeforeClass
+    public static void startSession() throws SAXException, ParserConfigurationException, jdk.internal.org.xml.sax.SAXException, IOException {
+        System.setProperty ("webdriver.chrome.driver", "C:\\SeleniumDrivers\\chromedriver.exe");
+        driver = new ChromeDriver ();
+        driver.get ("https://www.swiftic.com/");
+        driver.manage ().window ().maximize ();
+        driver.manage ().timeouts ().implicitlyWait (10, TimeUnit.SECONDS);
+    }
+
+    @Test
+    public void findElement() throws IOException, InterruptedException
+    {
+        driver.findElement (By.id ("menu-item-17422")).click ();
+
+        driver.findElement (By.id ("joinUserName") ).sendKeys ("testing@gmail.com" );
+
+        driver.findElement (By.cssSelector ("#registerForm > div:nth-child(2) > div > div > input") ).sendKeys ("testing@gmail.com" );
+
+        driver.findElement (By.id ("repassword") ).sendKeys ("testing@gmail1.com" );
+
+        driver.findElement (By.xpath ("//*[@id=\"registerForm\"]/div[4]/div/button")).click();
+
+        WebElement errormessage = driver.findElement (By.xpath ("//*[@id=\"registerForm\"]/div[7]/div/div/h6"));
+        Assert.assertEquals("Please re-enter your password", errormessage.getText ());
+
+    }
+
+    @AfterClass
+    public static void closeSession()
+    {
+//        driver.quit ();
+    }
+}
\ No newline at end of file
