Index: src/test/java/pages/mainPage.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package pages;\n\nimport org.junit.Test;\nimport org.openqa.selenium.WebDriver;\nimport org.openqa.selenium.WebElement;\nimport org.openqa.selenium.support.FindBy;\nimport org.openqa.selenium.support.How;\n\n\npublic class mainPage extends Utils.Base\n{\n    @FindBy(how = How.ID, using = \"menu-item-17422\")\n    public WebElement createbutton;\n\n    @FindBy(how = How.ID, using = \"chosen_language\")\n    public WebElement chosen_language;\n\n    @FindBy(how = How.ID, using = \"chosen_language\")\n    public WebElement language_1;\n\n    @FindBy(how = How.ID, using = \"chosen_language\")\n    public WebElement language_2;\n\n    @FindBy(how = How.ID, using = \"chosen_language\")\n    public WebElement language_3;\n\n    @FindBy(how = How.ID, using = \"chosen_language\")\n    public WebElement language_4;\n\n    @FindBy(how = How.ID, using = \"chosen_language\")\n    public WebElement language_5;\n\n}\n\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/pages/mainPage.java	(date 1530625732000)
+++ src/test/java/pages/mainPage.java	(date 1530697772886)
@@ -12,6 +12,7 @@
     @FindBy(how = How.ID, using = "menu-item-17422")
     public WebElement createbutton;
 
+
     @FindBy(how = How.ID, using = "chosen_language")
     public WebElement chosen_language;
 
