Index: src/test/java/Utils/Base.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package Utils;\nimport org.openqa.selenium.WebDriver;\n\n\npublic class Base\n{\n    protected WebDriver driver;\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/Utils/Base.java	(revision fc1c0e1808c27bff54aa862dae36a958ca9e3e55)
+++ src/test/java/Utils/Base.java	(date 1532467642188)
@@ -1,8 +1,15 @@
 package Utils;
 import org.openqa.selenium.WebDriver;
+import pages.FeaturesPage;
+import pages.MainPage;
+import pages.wizard;
 
 
 public class Base
 {
-    protected WebDriver driver;
+    protected static WebDriver driver;
+    protected static MainPage mp;
+    protected static wizard wz;
+    protected static FeaturesPage fp;
+
 }
